scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -173,6 +173,8 @@ else
     $newBuildCounter = $BuildNumber
     $VsTargetBranch = & dotnet msbuild $RepositoryPath\build\config.props /v:m /nologo /t:GetVsTargetBranch
     $NuGetSdkVsVersion = & dotnet msbuild $RepositoryPath\build\config.props /v:m /nologo /t:GetNuGetSdkVsSemanticVersion
+    $VsTargetChannel = & dotnet msbuild $RepositoryPath\build\config.props /v:m /nologo /t:GetVsTargetChannel
+    $VsTargetMajorVersion = & dotnet msbuild $RepositoryPath\build\config.props /v:m /nologo /t:GetVsTargetMajorVersion
```

Comment:
`buildinfo.json` now contains 2 additional properties `$VsTargetChannel` & `$VsTargetMajorVersion` which are used for generating VS bootstrapper in OptProf pipeline.